### PR TITLE
LPS-139097 Account Admin cannot view the account users assigned to the account

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

File: modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/retriever/test/AccountUserRetrieverTest.java [hunk 1]
```diff
@@ -257,6 +265,49 @@ public void testSearchAccountUsersWithPagination() throws Exception {
...
+				PermissionCheckerFactoryUtil.create(roleUser));
+
+			_assertSearch(null, users.size());
+		}
+		finally {
+			PermissionThreadLocal.setPermissionChecker(
+				originalPermissionChecker);
+		}
+	}
+
```
Comment (drewbrokke) [inline modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/retriever/test/AccountUserRetrieverTest.java:302]: @pei-jung there's a mismatch here between the method used to retrieve the accounts in the permission filter and the assertions here. It looks like this is only testing direct membership permissions, but in theory we would also be checking accounts that the user is associated with via Organization association. 

Let me know if I'm missing something.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

**Comment by pei-jung** [comment]
ci:test:sf

**Review by drewbrokke — APPROVED** [review]
LGTM 👍

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**